src/Fuel-Platform-Core/FLPlatform.class.st

```diff
@@ -107,13 +107,13 @@ FLPlatform class >> isResponsibleForCurrentPlatform [
 
 { #category : #'private-convenience' }
 FLPlatform class >> removeModifications [
-	((Smalltalk globals at: #(Smalltalk globals at: #PackageInfo)) named: self extensionProtocolName) extensionMethods do: [ :methodReference |
+
+	((Smalltalk globals at: #( Smalltalk globals at: #PackageInfo )) named: self extensionProtocolName) extensionMethods do: [ :methodReference |
```

Comment:
Is this code really used and functional?
`#( Smalltalk globals at: #PackageInfo )` is the literal array `#(#Smalltalk #globals #at: #PackageInfo)`, not a message send.
And it used as a key of a dictionary that usually contains symbols?

This is a lot for a monday :/

Reply:
This method is dead code so I'll just remove it :)